### DISCUSSION TITLE
Images boxes

### DISCUSSION THREAD

**Discussion by marianoibaldi** [quarto-dev/quarto-cli#3134] [Q&A]
I have a block of figures generated with a div block and I want to put a box around it and fill the box with some colour. The final docs will be html and pdf.

Is there a way to do that?

**Comment by mcanouil** [comment]
No idea right know about doing it for PDF.
What's certain is that the two formats require two different solutions.

For HTML, it's as "simple" as adding a CSS attributes:

```
::: {style="border: 2px solid black;"}
some random text
:::
```

See <https://www.w3schools.com/Css/css_boxmodel.asp> and related HTML/CSS topics to improve/tweak it to your needs.

**Reply by marianoibaldi** [reply]
Thank you, @mcanouil!
For the PDF part, I tried to use *mdframed* and *framed* Tex packages, but did not work. I think that maybe it cannot be used outside a div block

**Reply by mcanouil** [reply]
There are many ways in LaTeX to add boxes, but way less trivial than as for HTML.
